app/factory.py

```diff
@@ -43,23 +43,27 @@ def get_user(self, identifier):
             if rv is not None:
                 return rv
 
+user_datastore = DeploySQLAlchemyUserDatastore(db, User, Role)
```

Comment:
I think we need to keep the `user_datastore` instantiation to inside the `create_app` method, but I'd be fine with moving the `DeploySQLAlchemyUserDatastore` class definition to `models.py`.